deploy/docs/HelmChartConfigiuration.md

```diff
@@ -0,0 +1,166 @@
+# Configuration
```

Comment:
once #683 is merged, will need to add `sumologic.logs.enabled` , `sumologic.metrics.enabled`, `fluentd.logd.enabled` and `fluentd.metrics.enabled`.

Reply:
I added the sumologic props, missed the fluentd props.  Will add those